x11-plugins/compiz-plugins-experimental/compiz-plugins-experimental-0.8.14.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils
+
+DESCRIPTION="Compiz Window Manager: Experimental Plugins"
+HOMEPAGE="https://github.com/compiz-reloaded"
+SRC_URI="https://github.com/compiz-reloaded/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	gnome-base/librsvg
```

Comment:
The following dependencies are missing here:
- media-libs/glew:=
- net-misc/curl
- x11-libs/cairo[X]
- x11-libs/libXScrnSaver
- x11-plugins/compiz-plugins-extra